server/lib/payments.js

```diff
@@ -468,3 +468,14 @@ export const sendReminderPendingOrderEmail = async order => {
     from: `${collective.name} <hello@${collective.slug}.opencollective.com>`,
   });
 };
+
+export const sendExpiringCreditCardUpdateEmail = async data => {
+  data = {
+    ...data,
+    updateDetailsLink: `${config.host.website}/${data.slug}/paymentmethod/${data.id}/update`,
+  };
+
+  return emailLib.send('payment.creditcard.expiring', data.email, data, {
+    from: `${data.slug} <hello@${data.slug}.opencollective.com>`,
```

Comment:
I think we're deprecating the _hello@collective_ email, we should check this with @Betree.

Reply:
In the end, I think we will need to write an utility function to compute this email address. In some cases, we will send `hello`, and in others it will be `no-reply`. We'll need to update everywhere it's used.

Reply:
#out-of-scope.

Reply:
Yes we'll make the changes with https://github.com/opencollective/opencollective/issues/2847, it's the standard today so I'm ok with using it here.

Reply:
Marking resolved since this is being addressed in another PR 👍